types/evidence_test.go

```diff
@@ -394,8 +324,8 @@ func TestEvidenceProto(t *testing.T) {
 		blockID := makeBlockID(tmhash.Sum([]byte("blockhash")), math.MaxInt32, tmhash.Sum([]byte("partshash")))
 		blockID2 := makeBlockID(tmhash.Sum([]byte("blockhash2")), math.MaxInt32, tmhash.Sum([]byte("partshash")))
 		const chainID = "mychain"
-		v := makeVote(t, val, chainID, math.MaxInt32, math.MaxInt64, 1, tmproto.PrevoteType, blockID, defaultVoteTime)
-		v2 := makeVote(t, val, chainID, math.MaxInt32, math.MaxInt64, 2, tmproto.PrevoteType, blockID2, defaultVoteTime)
+		v := makeVote(t, val, chainID, math.MaxInt32, math.MaxInt64, 1, 0x01, blockID, defaultVoteTime)
+		v2 := makeVote(t, val, chainID, math.MaxInt32, math.MaxInt64, 2, 0x01, blockID2, defaultVoteTime)
```

Comment:
Why hexadecimal?

Reply:
Would you see my thoughts here?:
* https://github.com/line/ostracon/pull/513/files#r1035624260

And give me your opinion there. Thanks.

Reply:
t looks like an implementation that has been around for a long time.
I don't feel the intention to be hex in particular, but I understood it as avoiding the feature backports.
I resolve this conversation. Thank you👍

https://github.com/tendermint/tendermint/commit/95badb088ae527d00f00e465509c2d7fe5c77c5d